auth/src/main/java/org/aerogear/auth/AbstractRole.java

```diff
@@ -22,4 +22,8 @@
     public String getRoleName() {
         return this.roleName;
     }
+
+    public String getClientID() {
```

Comment:
not sure if we need this on the base abstract class. I thought this is a specific field for client roles?

Reply:
+1